<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar_RPM">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>796</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>200</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>6000</number>
    </property>
    <property name="value">
     <number>3878</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="format">
     <string>%v rpm</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Vbat">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
    <property name="value" stdset="0">
     <double>112.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Imax">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>43.000000000000000</double>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_SOC">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>90</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="value">
     <number>78</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::BottomToTop</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_Tbat">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>90</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>67</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_Tdriver">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>90</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>85</number>
    </property>
    <property name="value">
     <number>68</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::BottomToTop</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_Tengine">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>90</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>47</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_SOC">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>400</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>78.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Tbat">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>400</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>67.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Tdriver">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>400</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>85.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Tengine">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>400</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>47.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Tmaxcell">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>400</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="numDigits">
     <number>4</number>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>113.650000000000006</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Tmincell">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="numDigits">
     <number>4</number>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>104.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>104</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Vmaxcell">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="numDigits">
     <number>4</number>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>113.650000000000006</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Vmincell">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>400</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="numDigits">
     <number>4</number>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>104.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>104</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_SOC">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>360</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SOC</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tengine">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>360</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>T ENG</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tdriver">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>360</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>T DRV</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tbat">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>360</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>T BAT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Vmincell">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>360</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>V MIN CELL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Vmaxcell">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>360</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>V MAX CELL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tmaxcell">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>T MAX CELL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tmincell">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>T MIN CELL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Vbat">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>V</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Imax">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_KMH">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
    <property name="value" stdset="0">
     <double>112.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_KMH">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>km/h</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_SOC_units">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Vmincell_units">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>V</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Vmaxcell_units">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>V</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tmincell_units">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>410</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tmaxcell_units">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>410</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
